src/variable.js

```diff
@@ -1,14 +1,26 @@
 import isWindows from 'is-windows'
 
+const pathLikeEnvVarWhitelist = ['PATH', 'NODE_PATH'].reduce(
```

Comment:
This seems overly complicated. What about simply `new Set(['PATH', 'NODE_PATH'])`?